### PR TITLE
acmp_134

### PR DESCRIPTION
**Pull request by Dirwul**
accepted

### CODE REVIEW

File: acmp/task_0134/solve.cpp [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+        if (cur.timeIn == INF) {
+            break;
+        }
+
+        for (const Way &way : ways[cur.id]) {
+            if (cur.timeIn <= way.timeStart && way.timeEnd < time[way.to]) {
+                time[way.to] = way.timeEnd;
+                q.push(City{ way.to, way.timeEnd });
+
```
Comment (terradrenor) [inline acmp/task_0134/solve.cpp]: remove empty line

File: acmp/task_0134/solve.cpp [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    }
+};
+
+struct City {
+
+    int id;
+
+    int timeIn;
+
+    bool operator<(const City &second) const{
```
Comment (terradrenor) [inline acmp/task_0134/solve.cpp]: add space after const

File: acmp/task_0134/solve.cpp (hunk 1 above)
Comment (Dirwul) [reply acmp/task_0134/solve.cpp]: done

File: acmp/task_0134/solve.cpp (hunk 2 above)
Comment (Dirwul) [reply acmp/task_0134/solve.cpp]: done

**Review by terradrenor — APPROVED** [review]

**Pull request merged**